src/persistence.rs

```diff
@@ -125,21 +118,6 @@ impl GossipPersister {
 						self.server_sync_completion_sender.send(()).await.unwrap();
 						println!("Server has been notified of persistence completion.");
 					}
-
-					// now, cache the persisted network graph
-					// also persist the network graph here
-					let mut too_soon = false;
```

Comment:
You sure you don't wanna cache the graph when we catch up?

Reply:
seems like a good checkpoint should it crash before the next sync happens, but maybe 10 minutes is frequent enough